lib/msal-browser/src/naa/TokenResponse.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AccountInfo } from "./AccountInfo";
+
+export type TokenResponse = {
```

Comment:
Reason for not using the existing AuthenticationResult type?

Reply:
Owned by bridge.